MinimedKit/GlucoseEvents/SensorTimestampGlucoseEvent.swift

```diff
@@ -8,11 +8,32 @@
 
 import Foundation
 
-public struct SensorTimestampGlucoseEvent: ReferenceTimestampedGlucoseEvent {
+public enum SensorTimestampType: String {
+    case lastRf
+    case pageEnd
+    case gap
+    case unknown
+    
+    public static func eventType(code: UInt8) -> SensorTimestampType {
```

Comment:
This would be more semantic as an initializer:
`public init(code: UInt8)`
(also it might not need to be public)